src/llv8.h

```diff
@@ -7,6 +7,14 @@
 
 #include "src/llv8-constants.h"
 
+using lldb::ByteOrder;
+
+#if __BYTE_ORDER == __LITTLE_ENDIAN
+#define HOST_BYTE_ORDER ByteOrder::eByteOrderLittle
+#else
+#define HOST_BYTE_ORDER ByteOrder::eByteOrderBig
+#endif
```

Comment:
Depending on `__BYTE_ORDER` is a little fragile, you often end up with a tower of #ifdefs to include the right file.  It's not necessary either because you can derive it at run-time:
```c++
inline bool IsBigEndian() {
  union { uint8_t a[2]; uint16_t b; } u = {{0, 1}};
  return u.b == 1;
}
```